aws/cloudformation/components/database.yml.erb

```
@@ -185,6 +186,51 @@
         # exceed 5% of the total database connection capacity of a reader instance.
         MaxConnectionsPercent: 5
 
+  WriterSQLUser: <%= lambda_fn 'SQLUser', {
+    # Wait until DB Instances have completed provisioning before using the admin SQL user to provision an application
+    # SQL user.
+    DependsOn: rack_env?(:production) ? nil : DB_INSTANCE_RANGE.map {|i| "Aurora#{i}"},
+    DBServerHost: {'Fn::GetAtt': ['AuroraCluster', 'Endpoint.Address']},
+    # Credentials (username and password) for the SQL User this Resource is provisioning.
```

is this supposed to be something like:
```suggestion
    # Credentials for SQL admin user that will create/update/delete this SQL user resource.
```
As in `ReaderSQLUser` below?